Spec/RealtimeClient.swift

```diff
@@ -333,6 +333,53 @@ class RealtimeClient: QuickSpec {
                     expect(secs).to(beLessThanOrEqualTo(UInt(options.suspendedRetryTimeout)))
                 }
             }
+
+            it("should never register any connection listeners for internal use with the public EventEmitter") {
```

Comment:
I've changed the code to _not_ use an internal event emitter and the tests as they are now were passing anyway, so I've gone ahead and implemented what I said. See 81033b8.